Added where can i find help? to faq (#84)
conda-incubator/conda-dot-org: learn/faq.md

## I see a problem with the website. How do I report it?

Please create an [issue](https://github.com/conda-incubator/conda-dot-org/issues) in GitHub to report any problems with the website. We also welcome pull requests!

## Where can I find help?

If you need help with any aspect of the conda ecosystem, feel free to reach out to us via any of our online channels.

> For more information read [How to reach us](https://conda-dot-org.netlify.app/community/#how-to-reach-us) on the [Community](https://conda-dot-org.netlify.app/community) page.

Reviewer: ```suggestion
For more information, read [How to reach us](https://conda-dot-org.netlify.app/community/#how-to-reach-us) on the [Community](https://conda-dot-org.netlify.app/community) page.
```

> For more information, read [How to reach us](https://conda-dot-org.netlify.app/community/#how-to-reach-us) on the [Community](https://conda-dot-org.netlify.app/community) page.

Reviewer: ```suggestion
For more information, read [How to reach us](/community/#how-to-reach-us) on the [Community](/community) page.
```

## I see a problem with the website. How do I report it?

Please create an [issue](https://github.com/conda-incubator/conda-dot-org/issues) in GitHub to report any problems with the website. We also welcome pull requests!

## Where can I find help?

> If you want some help you can reach out to us and get the necessary help.

Reviewer: ```suggestion
If you need help with any aspect of the conda ecosystem, feel free to reach out to us via any of our online channels.
```

> Here are the social profiles of conda : [Element](https://app.element.io/#/room/#conda_conda:gitter.im),  [Discourse](https://conda.discourse.group/), [Github](https://github.com/conda-incubator).

Reviewer: ```suggestion
```

> For more information [visit](http://localhost:3000/community#how-to-reach-us) here.

Reviewer: ```suggestion
For more information read [How to reach us](/community#how-to-reach-us) on the [Community](/community) page.
```
